examples/19.Generate_readonly_token.py

```diff
@@ -0,0 +1,33 @@
+from config import TEST_ACCT_KEY, TEST_NETWORK
```

Comment:
all test files have names in `lower_camel_case` please change this one's to follow the same